src/db/DbService.js

```
@@ -131,36 +123,32 @@ class DbService {
             queryParams.push(key, value, key, value, key, value, key, value);
         });
 
         if (address) {
-            query += ' AND user_wallet_address = ?';
+            query += (dataEntries.length ? 'AND' : '') + ' user_wallet_address = ?';
             queryParams.push(address);
         }
 
         if (excludeAttested) {
-            query += ' AND status != "attested"';
+            query += (dataEntries.length || address ? ' AND ' : '') + 'status != "attested"';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)